tests/test_glue/test_datacatalog.py

```diff
@@ -9,21 +9,36 @@
 
 from datetime import datetime
 import pytz
+from freezegun import freeze_time
 
 from moto import mock_glue
 from . import helpers
 
 
+FROZEN_CREATE_TIME = datetime(2015, 1, 1, 0, 0, 0)
+
+
 @mock_glue
+@freeze_time(FROZEN_CREATE_TIME)
 def test_create_database():
     client = boto3.client("glue", region_name="us-east-1")
     database_name = "myspecialdatabase"
-    helpers.create_database(client, database_name)
+    database_input = helpers.create_database_input(database_name)
+    helpers.create_database(client, database_name, database_input)
 
     response = helpers.get_database(client, database_name)
     database = response["Database"]
 
-    database.should.equal({"Name": database_name})
+    database.get("Name").should.equal(database_name)
+    database.get("Description").should.equal(database_input.get("Description"))
+    database.get("LocationUri").should.equal(database_input.get("LocationUri"))
+    database.get("Parameters").should.equal(database_input.get("Parameters"))
+    database.get("CreateTime").should.equal(FROZEN_CREATE_TIME)
```

Comment:
This assert doesn't work in ServerMode, as the server doesn't run inside the 'freeze_time' decorator.
You can make it conditional like this:
```
if not settings.TEST_SERVER_MODE:
database.get("CreateTime").should.equal(FROZEN_CREATE_TIME)
```